src/dbt/kipptaf/models/performance_management/intermediate/int_performance_management__overall_scores.sql

```diff
@@ -0,0 +1,165 @@
+            avg(etr_score) as etr_score,
+            avg(so_score) as so_score,
+            avg(overall_score) as overall_score,
+            case
+                when avg(etr_score) >= 3.5
+                then 4
+                when avg(etr_score) >= 2.745
+                then 3
+                when avg(etr_score) >= 1.745
+                then 2
+                when avg(etr_score) < 1.75
+                then 1
+            end as etr_tier,
+            case
+                when avg(so_score) >= 3.5
+                then 4
+                when avg(so_score) >= 2.945
+                then 3
+                when avg(so_score) >= 1.945
+                then 2
+                when avg(so_score) < 1.95
+                then 1
+            end as so_tier,
+            case
+                when avg(overall_score) >= 3.5
+                then 4
+                when avg(overall_score) >= 2.745
+                then 3
+                when avg(overall_score) >= 1.745
+                then 2
+                when avg(overall_score) < 1.75
+                then 1
+            end as overall_tier,
+            date(academic_year + 1, 5, 15) as eval_date,
+        from {{ ref("rpt_tableau__schoolmint_grow_observation_details") }}
+        where
+            form_type = 'PM'
+            and form_term in ('PM2', 'PM3')
+            and rn_submission = 1
+            and overall_score is not null
+        group by employee_number, academic_year
+    )
+
+select
+    s.employee_number,
+    s.academic_year,
+    s.pm_term,
+    s.etr_score,
+    s.so_score,
+    s.overall_score,
+    s.etr_tier,
+    s.so_tier,
+    s.overall_tier,
+    s.eval_date,
+
+    case
+        when sr.worker_termination_date is null
+        then
+            y.years_at_kipp_total
+            - ({{ var("current_academic_year") }} - s.academic_year)
+        else
+            y.years_at_kipp_total - (
+                {{
+                    teamster_utils.date_to_fiscal_year(
+                        date_field="worker_termination_date",
+                        start_month=7,
+                        year_source="start",
+                    )
+                }} - s.academic_year
+            )
+    end as years_at_kipp,
+
+    case
+        when sr.worker_termination_date is null
+        then
+            y.years_teaching_at_kipp
+            + sr.years_teaching_in_njfl
+            + sr.years_teaching_outside_njfl
+            - ({{ var("current_academic_year") }} - s.academic_year)
+        else
+            y.years_teaching_at_kipp
+            + sr.years_teaching_in_njfl
+            + sr.years_teaching_outside_njfl
+            - (
+                {{
+                    teamster_utils.date_to_fiscal_year(
+                        date_field="worker_termination_date",
+                        start_month=7,
+                        year_source="start",
+                    )
+                }} - s.academic_year
+            )
+    end as years_teaching,

```

Comment:
this should be calculated on `int_people__years_experience` and brought in at the reporting layer

Reply:
That case statement is attempting to determine how many years exp/teaching the person had for the given overall score year. ie: in 2018, when they scored a 3, they had 10 years experience; in 2019 it was 11, etc. Unless we change up the years experience model to be long by year, I think the calculation belongs on int_performance_managmenet__overall_scores